secretsmanager.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_secretsmanager_secret" "db_master_user" {
+  name = "ranger-db-master-user"
+}
```

Comment:
should add `tags = "${var.apiary_tags}"` to each secret

Reply:
done.